wpilibj/src/main/java/edu/wpi/first/wpilibj/IterativeRobotBase.java

```diff
@@ -100,7 +100,10 @@ protected IterativeRobotBase(double period) {
    * <p>Warning: the Driver Station "Robot Code" light and FMS "Robot Ready" indicators will be off
    * until RobotInit() exits. Code in RobotInit() that waits for enable will cause the robot to
    * never indicate that the code is ready, causing the robot to be bypassed in a match.
+   *
+   * @deprecated Use the robot class constructor instead.
```

Comment:
The method documentation should also be updated to explain that it's functionally identical to a constructor. The warning paragraph in particular stands out as outdated